src/requirements/data/majors/pam.ts

```diff
@@ -0,0 +1,226 @@
+  courseMatchesCodeOptions,
+  courseIsForeignLang,
+  includesWithSubRequirements,
+  courseMeetsCreditMinimum,
+} from '../checkers-common';
+
+const pamRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Core Courses',
+    description: 'PAM 2000, PAM 2030, PAM 2040, PAM 2101, PAM 2300, PAM 3100, and PAM 3300.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/PAM%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: includesWithSubRequirements(
+      ['PAM 2000'],
+      ['PAM 2030'],
+      ['PAM 2040'],
+      ['PAM 2101'],
+      ['PAM 2300'],
+      ['PAM 3100'],
+      ['PAM 3300']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1, 1, 1, 1, 1],
+    slotNames: ['PAM 2000', 'PAM 2030', 'PAM 2040', 'PAM 2101', 'PAM 2300', 'PAM 3100', 'PAM 3300'],
+  },
+  // TODO - limit this checker to only allow 1 course of PAM 4900 at a max of 3 credits
+  {
+    name: 'PAM 3000/4000 Electives',
+    description:
+      '18 credits of 3000 or 4000 level PAM courses. PAM 4000, 4010, 4020, 4030, 4060, 4980, and 4990 may not be used to fulfill this requirement, and a max of 3 credits from PAM 4900 can be used.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/PAM%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: [
+      (course: Course): boolean =>
+        ifCodeMatch(course.subject, 'PAM') &&
+        !courseMatchesCodeOptions(course, [
+          'PAM 4000',
+          'PAM 4010',
+          'PAM 4020',
+          'PAM 4030',
+          'PAM 4060',
+          'PAM 4980',
+          'PAM 4990',
+        ]) &&
+        (ifCodeMatch(course.catalogNbr, '3***') || ifCodeMatch(course.catalogNbr, '4***')),
+    ],
+    fulfilledBy: 'credits',
+    perSlotMinCount: [18],
+  },
+  {
+    name: 'Additional PAM Electives',
+    description: 'Any 3 additional PAM credits (except PAM 1200 & PAM 4030) or ECON 1120.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/PAM%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: [
+      (course: Course): boolean =>
+        (ifCodeMatch(course.subject, 'PAM') &&
+          !courseMatchesCodeOptions(course, ['PAM 1200', 'PAM 4030'])) ||
+        courseMatchesCodeOptions(course, ['ECON 1120']),
+    ],
+    fulfilledBy: 'credits',
+    perSlotMinCount: [3],
+  },
+  {
+    name: 'Introductory Microeconomics',
+    description: 'ECON 1110.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/PAM%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: includesWithSingleRequirement('ECON 1110'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Introduction to Sociology',
+    description: 'SOC 1101, DSOC 1101, or PAM 2250.',
```

Comment:
Can you add this sentence to the description: "Note: PAM 2250 cannot count for both the Introduction to Sociology requirement AND the Additional PAM Electives requirement."

Reply:
I didn't add this because neither requirement is double countable, so CoursePlan would not let users count PAM 2250 for both even if users tried. Do you still think I should add it?